tests/unit/core/test_deploy_export.py

```diff
@@ -350,3 +341,28 @@ def test_hf_bert(self, tmpdir, df_type):
         tmp_file_name = str(tmpdir.mkdir("export").join("bert"))
         # Test export.
         self.__test_export_route(module=bert, out_name=tmp_file_name, mode=df_type, input_example=input_example)
+
+    @pytest.mark.unit
+    @pytest.mark.run_only_on('GPU')
+    @pytest.mark.parametrize("df_type", [DF.ONNX, DF.TORCHSCRIPT, DF.PYTORCH])
+    def test_waveglow(self, tmpdir, df_type):
+        url = "https://api.ngc.nvidia.com/v2/models/nvidia/waveglow_ljspeech/versions/2/files/WaveGlowNM.pt"
+        ptfile = "./WaveGlowNM.pt"
+        if not Path(ptfile).is_file():
+            urllib.request.urlretrieve(url, ptfile)
+
+        module = nemo_tts.WaveGlowInferNM(sample_rate=22050)
+        module.restore_from(ptfile)
+        module.eval()
+
+        torch.manual_seed(1)
+        mel = torch.randn(1, 80, 96).cuda()
+        stride = 256  # value from waveglow upsample
+        n_group = 8
+        z_size2 = (mel.size(2) * stride) // n_group
+        z = torch.randn(1, z_size2).cuda()
+
+        input_example = OrderedDict([("mel_spectrogram", mel), ("z", z)])
+        tmp_file_name = str(tmpdir.mkdir("export").join("waveglow"))
+
+        self.__test_export_route(module=module, out_name=tmp_file_name, mode=df_type, input_example=input_example)
```

Comment:
I am now very confused by this PR. Are you sure that you are exporting the inference route when you do this?